### ISSUE TITLE
Bundling with Webpack fails

### ISSUE THREAD

**Issue by JimiC** [danvk/dygraphs#847]
I'm using `dygraphs` on an Angular 2 project via `angular-cli`. When I try to bundle the project for production (it uses `webpack`), the bundling fails cause it tries to bundle the `es6` version.

I found out that if I change in the `package.json` the `module` property to `index.es5` everything is fine.
I would have submitted a PR with this change but I'm not sure if this will break things for others.

Please advise.

**Comment by mathisloge** [comment]
+1 same here. 
After switching module to index.es5 it works

**Comment by stefan-leye** [comment]
+1

**Comment by mleprince** [comment]
:+1: 
I've set the "module" property in package.json to "index.es5" and it doesn't work.
@JimiC : Can you show me you package.json please ?

**Comment by JimiC** [comment]
@mleprince You need to set this project's `package.json` `module` property to `index.es5`.

**Comment by opavader** [comment]
In my case webpack is failing on `Uglify` step as UglifyJS doesn't support `es6` syntax. Should be similar reason for others.

**Comment by danvk** [comment]
Take a look at the [dygraphs-es6 repo](https://github.com/danvk/dygraphs-es6). It includes examples of bundling dygraphs with webpack and rollup.

**Comment by JimiC** [comment]
@danvk Excuse but the topic here isn't the use with webpack directly but the issue we have when using it with Angular which uses webpack (note here that Angular doesn't provide the ability to configure webpack).
I would really appreciate if you take the time and examine my proposal.

**Comment by danvk** [comment]
If you can make that change without breaking the dygraphs-es6 repo, I'd be happy to consider it.

When I was setting up the dygraphs `package.json` (#815) I used modules like d3-scale as a template. Its `module` property points at a file which uses ES6 imports. If it works with Angular, I'd be curious to know what the difference between the two is.

**Comment by JimiC** [comment]
@danvk I know now more about the `module` property than I wanted. As I suspected changing that property's value is not a viable solution. I'll investigate this further.

**Comment by JimiC** [comment]
This issue is related to https://github.com/mishoo/UglifyJS2/issues/448

**Comment by opavader** [comment]
@JimiC I agree. A [workaround](https://github.com/mishoo/UglifyJS2#harmony) if anyone wants to use the es6 version.
